### PR TITLE
Add CVE-2022-23638.

### PR DESCRIPTION
**Pull request by beloglazov91**
Add CVE-2022-23638.

### CODE REVIEW

File: enshrined/svg-sanitize/CVE-2022-23638.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     A cross-site scripting vulnerability
+link:      https://github.com/darylldoyle/svg-sanitizer/releases/tag/0.15.0
+cve:       CVE-2022-23638
+branches:
+    master:
+        time:     2022-01-19 12:00:00
```
Comment (sjerdo) [inline enshrined/svg-sanitize/CVE-2022-23638.yaml:6]:
```suggestion
        time:     2021-10-11 23:08:26
```

Comment (naderman) [reply enshrined/svg-sanitize/CVE-2022-23638.yaml:6]: Neither this nor the original time seem accurate? It looks like the fix was merged `2022-02-13 01:42:00`?

Comment (naderman) [reply enshrined/svg-sanitize/CVE-2022-23638.yaml:6]: refering to time of this merge commit: https://github.com/darylldoyle/svg-sanitizer/commit/17e12ba9c2881caa6b167d0fbea555c11207fbb0

**Comment by naderman** [comment]
@beloglazov91 Can you explain where that date is coming from? See https://github.com/FriendsOfPHP/security-advisories/pull/610#discussion_r808828309

File: enshrined/svg-sanitize/CVE-2022-23638.yaml (hunk 1 above)
Comment (beloglazov91) [reply enshrined/svg-sanitize/CVE-2022-23638.yaml]: @naderman applied that time.

File: enshrined/svg-sanitize/CVE-2022-23638.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title:     A cross-site scripting vulnerability
+link:      https://github.com/darylldoyle/svg-sanitizer/releases/tag/0.15.0
```
Comment (naderman) [inline enshrined/svg-sanitize/CVE-2022-23638.yaml:2]:
```suggestion
link:      https://github.com/advisories/GHSA-fqx8-v33p-4qcc
```

Sorry one more thing, link should go to security announcement, not tag for the fix

**Comment by beloglazov91** [comment]
@naderman Let's as a date use the published date of `https://github.com/advisories/GHSA-fqx8-v33p-4qcc` (2022-02-15 01:54:00). I've pushed my changes.

**Pull request merged**